packages/0xcert-ethereum-gateway-contracts/src/contracts/actions-gateway.sol

```diff
@@ -290,15 +275,21 @@ contract ActionsGateway is
   )
     public
   {
-    require(_data.maker == msg.sender, MAKER_NOT_EQUAL_TO_SENDER);
+    bool present = false;
+    for (uint8 i = 0; i < _data.signers.length; i++) {
+      if (_data.signers[i] == msg.sender) {
+        present = true;
+        break;
+      }
+    }
+    require(present, MAKER_NOT_EQUAL_TO_SENDER);
 
     bytes32 claim = getOrderDataClaim(_data);
     require(!orderPerformed[claim], ORDER_ALREADY_PERFORMED);
 
     orderCancelled[claim] = true;
     emit Cancel(
-      _data.maker,
-      _data.taker,
+      _data.signers,
```

Comment:
This can be expensive. Review if this is necessary because it is putting an array in the event log.